microshift_support/microshift-etcd.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="etcd-microshift"]
+= MicroShift etcd systemd log stream
+//another name idea could be "Etcd as a transient systemd unit that is managed by MicroShift"
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
+:context: etcd-microshift
+
+toc::[]
+
+MicroShift etcd logs will operate in their own system log stream by running the etcd container image under Podman and running the etcd binary installed on a MicroShift RPM.
+
+:FeatureName: MicroShift
+include::snippets/technology-preview.adoc[leveloffset=+1]
+//
+
+As a part of MicroShift installation, etcd will automatically start by using `systemd-run etcd`. You will be able to view etcd running under systemd `systemctl status microshift-etcd` and collect logs from it.
```

Comment:
The MicroShift process manages the etcd process. How it does that is an implementation detail, so let's leave that out of these docs.

```suggestion
MicroShift manages the etcd process automatically. You will be able to view etcd running under systemd `systemctl status microshift-etcd` and collect logs from it.
```